space_packet_parser/parseables.py

```diff
@@ -104,15 +94,44 @@ def read_as_int(self, nbits: int) -> int:
         : int
             Integer representation of the bits read from the packet
         """
-        int_data = _extract_bits(self.rawdata, self.pos, nbits)
+        int_data = _extract_bits(self, self.pos, nbits)
         self.pos += nbits
         return int_data
 
 
+class Packet(dict):
+    """CCSDS Packet
+
+    Container that stores the raw packet data (bytes) and the parsed
+    data items. The raw packet data is stored as a ``RawPacketData`` object
+    that keeps track of the current parsing position.
```

Comment:
I think we should add a comment that mentions that self.items gets populated when the packet is parsed.